annotationPipeline/src/main/java/org/cbioportal/annotation/AnnotationPipeline.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2016 Memorial Sloan-Kettering Cancer Center.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY, WITHOUT EVEN THE IMPLIED WARRANTY OF MERCHANTABILITY OR FITNESS
+ * FOR A PARTICULAR PURPOSE. The software and documentation provided hereunder
+ * is on an "as is" basis, and Memorial Sloan-Kettering Cancer Center has no
+ * obligations to provide maintenance, support, updates, enhancements or
+ * modifications. In no event shall Memorial Sloan-Kettering Cancer Center be
+ * liable to any party for direct, indirect, special, incidental or
+ * consequential damages, including lost profits, arising out of the use of this
+ * software and its documentation, even if Memorial Sloan-Kettering Cancer
+ * Center has been advised of the possibility of such damage.
+ */
+
+/*
+ * This file is part of cBioPortal CMO-Pipelines.
+ *
+ * cBioPortal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+package org.cbioportal.annotation;
+
+import org.cbioportal.annotation.pipeline.BatchConfiguration;
+
+import org.apache.commons.cli.*;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.batch.core.*;
+import org.springframework.batch.core.launch.JobLauncher;
+
+/**
+ * @author Zachary Heins
+ */
+@SpringBootApplication
+public class AnnotationPipeline
+{
+
+    private static Options getOptions(String[] args)
+    {
+        Options gnuOptions = new Options();
+        gnuOptions.addOption("h", "help", false, "shows this help document and quits.")
+            .addOption("f", "filename", true, "Mutation filename")
+            .addOption("d", "directory", true, "Output directory")
```

Comment:
I think directory should be an output filename (which can include the path).  This simplifies the code in the MutationRecordWriter and you don't need to capture a genomenexus.output_filename property (which is not very flexible).